### PR TITLE
[new][author] Add react-highcharts@5.0.6

### PR DESCRIPTION
**Pull request by kirjs**
Docs: https://github.com/kirjs/react-highcharts

### CODE REVIEW

**Comment by kirjs** [comment]
See #6317

File: ajax/libs/react-highcharts/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "react",
+    "highcharts",
+    "graph"
+  ],
+  "readmeFilename": "README.md",
+  "gitHead": "896bf22728ae74749d3355695adb2fee63c921af",
+  "homepage": "https://github.com/kirjs/react-highcharts#readme",
+  "_id": "react-highcharts@5.0.6",
+  "_shasum": "815331d88d9f0c1b8668fcebd3fc6d9b79067bb1",
+  "_from": "react-highcharts@",
```
Comment (PeterDaveHello) [inline ajax/libs/react-highcharts/package.json]: Please remove line 5, 24~29, we don't need them, thanks.

Comment (kirjs) [inline ajax/libs/react-highcharts/package.json]: done

**Comment by PeterDaveHello** [comment]
Ok, thanks, I have some questions here:

 - Don't you need `bundle` dir under `dist`?
 - Personally, I think highchart, highmaps and hightstock are different products, does it make sense to include all of them in react-highcharts? 

Thanks.

**Comment by kirjs** [comment]
1. Bundles are just helpers which include Highcharts code which is already on cdnjs, so I'd avoid duplication and not include them. 
2. They historically got into the same repository, because due to complex Highcharts module structure it was easier to support them this way. I would make sense to separate them out at some point, but currently this is where the users would expect those modules to be.

**Comment by PeterDaveHello** [comment]
Thanks for your explanation @kirjs

**Comment by kirjs** [comment]
Thanks a lot!

**Pull request merged**